test/src/test/java/org/corfudb/protocols/logprotocol/MultiObjectSMREntryTest.java

```diff
@@ -0,0 +1,94 @@
+package org.corfudb.protocols.logprotocol;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import org.corfudb.CustomSerializer;
+import org.corfudb.runtime.exceptions.SerializerException;
+import org.corfudb.util.serializer.ISerializer;
+import org.corfudb.util.serializer.Serializers;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class MultiObjectSMREntryTest {
+
+    @Test
+    public void testLazyDeserialization() {
```

Comment:
This is a nice test, can we add a summary of what the test is exercising?

Reply:
Done.